HIP/hip-0000-event-serialization.md

```diff
@@ -0,0 +1,227 @@
++------------+-----------------------+---------------------------------------------+---------------+
+| 4 bytes    | Integer               | Serialization version of object             | 0x00000004    |
++------------+-----------------------+---------------------------------------------+---------------+
+| V bytes    | SoftwareVersion       | The software version that created the event |               |
++------------+-----------------------+---------------------------------------------+---------------+
+| 12 bytes   | NodeId                | The node that created the event             |               |
++------------+-----------------------+---------------------------------------------+---------------+
+| S bytes    | EventDescriptor       | The descriptor of the event's self parent   |               |
++------------+-----------------------+---------------------------------------------+---------------+
+| L bytes    | EventDescriptor[]     | A list of descriptors for other parents     |               |
++------------+-----------------------+---------------------------------------------+---------------+
+| 8 bytes    | Long                  | The round the event was created             |               |
++------------+-----------------------+---------------------------------------------+---------------+
+| I bytes    | Instant               | The time the event was created              |               |
++------------+-----------------------+---------------------------------------------+---------------+
+| T bytes    | Transaction[]         | The transactions within the event           |               |
++------------+-----------------------+---------------------------------------------+---------------+
+```
+
+### BaseEvent
+
+```
++--------------------------------------------------------------------------------------------------+
+| BaseEvent Object                                                                                 |
++------------+-----------------------+---------------------------------------------+---------------+
+| # of bytes | Type                  | Description                                 | Typical Value |
++------------+-----------------------+---------------------------------------------+---------------+
+| 4 bytes    | Integer               | Serialization version of object             | 0x00000004    |
++------------+-----------------------+---------------------------------------------+---------------+
+| H bytes    | BaseEventHashedData   | The hashed data of the event                |               |
++------------+-----------------------+---------------------------------------------+---------------+
+| 4 bytes    | Integer               | The length of the signature = S             |               |
++------------+-----------------------+---------------------------------------------+---------------+
+| S bytes    | Byte[]                | The signature of the node signing the event |               |
```

Comment:
You've already used `S` to denote number of bytes in the `EventDescriptor`. Not sure if that matters

Reply:
Changed event descriptor to D.